src/main/java/net/earthcomputer/multiconnect/protocols/v1_15_2/Protocol_1_15_2.java

```diff
@@ -522,9 +522,13 @@ private void mutateBlockRegistry(RegistryBuilder<Block> registry) {
         registry.unregister(Blocks.CHISELED_NETHER_BRICKS);
         registry.unregister(Blocks.CRACKED_NETHER_BRICKS);
         registry.unregister(Blocks.QUARTZ_BRICKS);
+
     }
 
     private void mutateItemRegistry(RegistryBuilder<Item> registry) {
+        //registry.unregister(Items.POWDER_SNOW_BUCKET);
+        //registry.unregister(Items.GLOW_BERRIES);
+        //registry.unregister(Items.GLOW_INK_SAC);
```

Comment:
Probably leftover

Reply:
Yes ik It was debug it will removed when the full version is released